Sources/Graphiti/Types/UnionType.swift

```diff
@@ -0,0 +1,8 @@
+import GraphQL
+
+public final class UnionTypeBuilder<Type> {
+    public var description: String? = nil
+    public var resolveType: GraphQLTypeResolve? = nil
+    public var types: [Any.Type] = []
```

Comment:
I tried using `[Type.Type]` here (and in the `union<Type>` function `members` arguments) in an attempt to make it a bit more type safe however I was then unable to actually produce any code that would compile without errors like... `Cannot convert value of type ... to expected argument type '[_.Type]'`